metricbeat/module/elasticsearch/testing.go

```diff
@@ -0,0 +1,29 @@
+package elasticsearch
+
+import "os"
+
+func GetEnvHost() string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function GetEnvHost should have comment or be unexported